.circleci/continue-config.yml

```diff
@@ -58,12 +58,7 @@ jobs:
       - vfcommon/clone_s3_assets:
           step_name: Uploading React Chat to CDN
           from: packages/react-chat/dist
-          to: s3://cdn.voiceflow.com/react-chat
-          clean_destination: false
-      - vfcommon/clone_s3_assets:
-          step_name: Uploading Widget to CDN
-          from: packages/widget/dist
-          to: s3://cdn.voiceflow.com/widget
+          to: s3://cdn.voiceflow.com/widget/alpha # TODO: change to production after validation
```

Comment:
should be changed back now?

Reply:
I want to still push it here to test the CI/CD pipeline. After the first push on master works, I'll switch it back.